src/wallet/rpcdump.cpp

```diff
@@ -217,6 +221,29 @@ Value importaddress(const Array& params, bool fHelp)
     return Value::null;
 }
 
+Value z_importwallet(const Array& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return Value::null;
+
+    if (fHelp || params.size() != 1)
+        throw runtime_error(
+            "z_importwallet \"filename\"\n"
```

Comment:
The rationale is that we plan to deprecate the entire Bitcoin RPC interface moving forward. Meanwhile, for the 1.0 launch, we want it to behave as identically to Bitcoin as possible.

This is dangerous, though. We need to add clear warnings to the live help strings for these methods that they exclude the zkeys. (@bitcartel tells me we've already done this.)